pkg/controller/decorator/controller.go

```diff
@@ -19,6 +19,7 @@ package decorator
 import (
 	"context"
 	"fmt"
+	v12 "metacontroller/pkg/controller/decorator/api/v1"
```

Comment:
Is this typo? Why v12?

Reply:
yes, goland didi it as `v1` was taken, fixed